### DISCUSSION TITLE
Environment Setting for Match Detection

### DISCUSSION THREAD

**Discussion by DaveSH12** [dani-garcia/vaultwarden#4986] [Q&A] [answered]
Hey Community,
Can I set the default Setting for Match Detection in the docker compose file? I didn't find any informations for this.
Thanks.

Kind regards,
David

**Comment by stefan0xC** [comment]
No, this is a client side setting so it's not possible. (At least not without building a patched version yourself as #3748 has been rejected.)

**Reply by DaveSH12** [reply]
Sorry, I think we have a misunderstanding here. When I create a new item, there is a field called 'Match detection' when setting URI 1, and the default value here is always set to 'Default match detection'. How can I determine the default value for my Vaultwarden server?

**Reply by stefan0xC** [reply, marked as answer]
It's not something you can control server side. That's what the PR would have been for.
